lib/elixir_analyzer/test_suite/leap.ex

```diff
@@ -0,0 +1,31 @@
+defmodule ElixirAnalyzer.TestSuite.Leap do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Leap
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+
+  assert_no_call "does not call any functions from :calendar Erlang module" do
+    type :essential
+    called_fn module: :calendar, name: :_
+    comment ElixirAnalyzer.Constants.leap_erlang_calendar()
+  end
+
+  feature "does not alias or import :calendar" do
```

Comment:
This feature is not necessary anymore, `assert_call` automatically keeps track of imports and aliases.

Reply:
Resolved in 33c4e08.